kconfigs/Kconfig.features

```diff
@@ -298,6 +298,11 @@ config HAS_PERIPH_TIMER_PERIODIC
         Indicates that the Timer peripheral provides the periodic timeout
         functionality.
 
+config HAS_PERIPH_TIMER_QUERY_FREQ
```

Comment:
I believe this should also have an associated `MODULE_PERIPH_TMER_QUERY_FREQ` which depends on this feature, right?

Reply:
Thanks. Should be fixed now.